.travis.yml

```diff
@@ -56,7 +56,7 @@ matrix:
             - ubuntu-toolchain-r-test
 
     # gcc-7
-    - env: VERSION=7 CXX_VERSIONS="11;14;17"
+    - env: VERSION=7 CXX_VERSIONS="11;14;17" CXXFLAGS="-Wno-noexcept-type"
```

Comment:
I don't recall what this was for... why do we need this?

Reply:
GCC 7.x warns in 11;14 modes that a mangled name will change if compiled for 17 due to noexcept in the type system. The warning was bogus, the check couldn't be implemented properly, and it got removed in GCC 8.